packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -28,19 +28,19 @@ OAuth 2.0 is a standard that apps use to provide client applications with access
 
 The OAuth 2.0 spec has four important roles:
 
-* The "authorization server" &mdash; The server that issues the access token. In this case Okta is the authorization server.
+* **authorization server**: The server that issues the access token. In this case Okta is the authorization server.
 
-* The "resource owner" &mdash; Normally your application's end user that grants permission to access the resource server with an access token
+* **resource owner**: Normally your application's end user that grants permission to access the resource server with an access token.
 
-* The "client" &mdash; The application that requests the access token from Okta and then passes it to the resource server
+* **client**: The application that requests the access token from Okta and then passes it to the resource server.
 
-* The "resource server" &mdash; Accepts the access token and must verify that it's valid. In this case this is your application.
+* **resource server**: Accepts the access token and must verify that it's valid. In this case, this is your application.
```

Comment:
```suggestion
* **resource server**: Accepts the access token and must verify that it's valid (typically, this is your application)
```

Reply:
As discussed over Slack will stick with the period.